python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -646,6 +646,10 @@ def get_asset_deps(
     Dict[str, List[str]],
     Dict[str, Dict[str, Any]],
 ]:
+    from .dagster_dbt_translator import validate_translator
+
+    dagster_dbt_translator = validate_translator(dagster_dbt_translator)
```

Comment:
Put after the imports

Reply:
this is a cyclic import so it  can't go at the top level

Reply:
I meant to put this line after all the inline imports. But nbd.